tests/test_testclient.py

```diff
@@ -132,3 +146,20 @@ async def asgi(receive, send):
     with client.websocket_connect("/") as websocket:
         data = websocket.receive_json()
         assert data == {"message": "test"}
+
+
+@pytest.mark.parametrize("endpoint", ["/slow_response", "/async_slow_response"])
+def test_timeout(endpoint):
+    client = TestClient(mock_service, raise_server_exceptions=True)
```

Comment:
Should we test with `raise_server_exceptions=False`?

Reply:
Default is `raise_server_exceptions=True`. 

I should perhaps add a test (check for 500 status code) when set to `False`...